website/content/docs/concepts/lease.mdx

```diff
@@ -68,6 +68,9 @@ so often.
 As a result, the return value of renewals should be carefully inspected to
 determine what the new lease is.
 
+
+To implement token renewal logic in your application code, refer to the [code example in the Authentication doc](/docs/concepts/auth#code-example).
```

Comment:
One last thing...  Let's put this in a call out box.

```suggestion
-> To implement token renewal logic in your application code, refer to the [code example in the Authentication doc](/docs/concepts/auth#code-example).
```

Reply:
ok